api/src/functionalTest/java/uk/gov/hmcts/payment/functional/idam/IdamApi.java

```diff
@@ -29,7 +29,7 @@ AuthenticateUserResponse authenticateUser(@Param("authorization") String authori
                                               @Param("client_id") String clientId,
                                               @Param("redirect_uri") String redirectUri);
 
```

Comment:
Please refer to https://github.com/hmcts/ccpay-bulkscanning-app/commit/f6437a25140cb7cafd30b711741276ecc45195f2, need some code clean up

Reply:
Changes done.